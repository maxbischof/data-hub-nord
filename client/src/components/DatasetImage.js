import React from 'react'
import styled from 'styled-components'
//import PropTypes from 'prop-types'

export default function DatasetImage({ path }) {
  return <Image src={path} />
}

const Image = styled.img`
  border-radius: 5px;
  border: 1px solid var(--grey);
  width: 100%;
  margin: 0;
`

// DatasetImage.propTypes = {
//   path: PropTypes.string.isRequired,
// }
