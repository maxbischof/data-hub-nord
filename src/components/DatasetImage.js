import React from 'react'
import styled from 'styled-components'

export default function DatasetImage({ path }) {
  return <Image src={path}/>
}

const Image = styled.img`
  border-radius: 5px;
  border: 1px solid var(--grey);
  width: 100%;
  margin: 0 0 0 0;
`