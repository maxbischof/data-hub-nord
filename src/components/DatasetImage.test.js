import React from "react"
import DatasetImage from "./DatasetImage"
import renderer from "react-test-renderer"

it("renders datasetimage component", () => {
  const tree = renderer
    .create(<DatasetImage path="../images/sea.jpg" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
