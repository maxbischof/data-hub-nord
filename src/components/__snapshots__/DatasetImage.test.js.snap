// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders datasetimage component 1`] = `
<img
  className="sc-AxjAm dqGSZK"
  src="../images/sea.jpg"
/>
`;
